src/ontology/cl-edit.owl

```diff
@@ -32238,6 +32248,65 @@ SubClassOf(obo:CL_4042016 obo:CL_0000313)
 SubClassOf(obo:CL_4042016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005385))
 SubClassOf(obo:CL_4042016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0011148))
 
+# Class: obo:CL_4042017 (α1-tanycyte)
```

Comment:
IIRC, greek letters should not be used (use 'alpha' instead). I can't find it in the editing guidelines, @addiehl am I correct?

Reply:
Probably included in:
Avoid special characters: use only alphanumeric characters, space, dash (-), slash (/), and apostrophe (').

Reply:
Can I use the term α1-tanycyte as a synonym instead?

Reply:
I don't think so, I searched for it and I can't find any greek letter in CL.

Reply:
The synonyms are still there.

Reply:
ok let me delete them